### ISSUE TITLE
act.jpg有把文字分割，但是画线却画不出

### ISSUE THREAD

**Issue by Owenscu** [huoyijie/AdvancedEAST#61]
![image](https://user-images.githubusercontent.com/23129979/54606422-c0122b00-4a86-11e9-8c00-6c4abe5cd629.png)
![image](https://user-images.githubusercontent.com/23129979/54606436-c6a0a280-4a86-11e9-85a8-8caa43d7e318.png)

**Comment by hcnhatnam** [comment]
You can try increasing **trunc_threshold** in cfg.py

**Comment by Owenscu** [comment]
@hcnhatnam thanks, i will try it.

**Comment by hcnhatnam** [comment]
@Owenscu  Can you tell me what training you have on the dataset and what is the **valid loss** value after the training?

**Comment by Owenscu** [comment]
@hcnhatnam on my dataset, val_loss is 0.35

**Comment by hcnhatnam** [comment]
what is the valid loss value on epoch 1 and where epoch drops to 0.35?

**Comment by Owenscu** [comment]
@hcnhatnam it seems that is something wrong.
![image](https://user-images.githubusercontent.com/23129979/54681325-029b3c80-4b47-11e9-94a8-004736b39819.png)

**Comment by hcnhatnam** [comment]
From epoch number 18 and after, this model is overfit?what is the valid loss value on epoch 1 ?

**Comment by Owenscu** [comment]
@hcnhatnam 0.59

**Comment by hcnhatnam** [comment]
oh...Your loss has decreased insignificantly....Did you train on any size [256, 384, 512, 640, 736]?

**Comment by Owenscu** [comment]
@hcnhatnam no. I will try when i am free. thanks

**Comment by hcnhatnam** [comment]
Ok @Owenscu .I still don't understand why the valid loss is **still so high**? I wonder if it's because the code is mistaken somewhere @huoyijie

**Comment by Owenscu** [comment]
@hcnhatnam enen, i am checking.

**Comment by zdyshine** [comment]
我成功运行了程序、想咨询一下，分阶段训练[256, 384, 512, 640, 736]的epoch问题。在比如在尺寸256，epoch=16的情况下，保存了最优模型。我在下一尺寸384，该怎么设置起始的epoch？我在后面也看到了回答，但是没看太懂，我的理解是下一阶段的epoch=17.请问是这样吗？期待回复。

**Comment by hcnhatnam** [comment]
It is not important to set the next epoch to 17 or not.

**Comment by clw5180** [comment]
> @hcnhatnam enen, i am checking.

华为书法识别？留个联系方式交流下吧

**Comment by clw5180** [comment]
> @hcnhatnam enen, i am checking.

华为书法识别？留个联系方式交流下吧

**Comment by zdyshine** [comment]
老哥、初赛马上就结束了啊。。。我Q：690866816。可讨论下




| |
张东阳
学生
|
|
江西财经大学
17607086525
邮箱：zdysuyi@163.com
江西省南昌市江西财经大学
690866816
海阔凭鱼跃 天高任鸟飞
|

签名由 网易邮箱大师 定制

在2019年03月24日 00:44，Henry 写道：

@hcnhatnam enen, i am checking.

华为书法识别？留个联系方式交流下吧

**Comment by Owenscu** [comment]
@zdyshine 我不是搞cv的，而且做这个比赛也比较晚，成绩不是很好，你还需要交流嘛，需要的话我就加你。

**Comment by zdyshine** [comment]
学习第一、比赛第二～～




| |
张东阳
学生
|
|
江西财经大学
17607086525
邮箱：zdysuyi@163.com
江西省南昌市江西财经大学
690866816
海阔凭鱼跃 天高任鸟飞
|

签名由 网易邮箱大师 定制

在2019年03月24日 11:34，Owenscu 写道：

@zdyshine 我不是搞cv的，而且做这个比赛也比较晚，成绩不是很好，你还需要交流嘛，需要的话我就加你。

**Comment by Owenscu** [comment]
@zdyshine 690866816这是你qq嘛

**Comment by snowwindy** [comment]
你好，我想问一下，Trunc_threshold这个阈值是啥意思呀？没搞清楚

**Comment by zhanshenlc** [comment]
> 我成功运行了程序、想咨询一下，分阶段训练[256, 384, 512, 640, 736]的epoch问题。在比如在尺寸256，epoch=16的情况下，保存了最优模型。我在下一尺寸384，该怎么设置起始的epoch？我在后面也看到了回答，但是没看太懂，我的理解是下一阶段的epoch=17.请问是这样吗？期待回复。

@zdyshine Set initial_epoch to 16, it will automatically increment this by one.

**Comment by cuimiao187561** [comment]
我检测长文本遇到同样问题，好多激活图（act.jpg）没有绿色和黄色边界点 @Owenscu 
![013 jpg_act](https://user-images.githubusercontent.com/23358104/60314917-008f1800-9997-11e9-9cbb-fda2eb68fd3a.jpg)

**Comment by www516717402** [comment]
@cuimiao187561 Firstly try to increase trunc_threshold paramter. Secondly try finetune net by author provide method. finally you can modify nms strategy such as supplement by manual.

**Comment by xiiiiiiii** [comment]
> 我检测长文本遇到同样问题，好多激活图（act.jpg）没有绿色和黄色边界点 @Owenscu
> ![013 jpg_act](https://user-images.githubusercontent.com/23358104/60314917-008f1800-9997-11e9-9cbb-fda2eb68fd3a.jpg)

你好，想问问你有头没尾或者有尾无头的问题是怎么解决的？或者有什么参考的方法吗？

**Comment by v-smwang** [comment]
+1
